char_index.py

```diff
@@ -0,0 +1,29 @@
+char_position = 0
+lowecase_chars = ['countat1','a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o', 'p', 'q', 'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z']
+return_message = "Position of alphabet: {char_position}"
+character_argument_max_len = 1
+    
+def position(character_argument: str) -> str:
+    """with a letter, return its position in the alphabet
+    
+    Args:
+    String, a single character 
+    
+    Returns
+    String, a string in the format of 'Position of alphabet: char_position'
+    
+    Raises
+    ca
+    """
+    
+    # Sanity Checks
+    if not (isinstance(character_argument, str) and len(character_argument) == character_argument_max_len):
+        raise ValueError("You're SOOO stoooodpid.")
+        
+    if character_argument not in lowecase_chars:
+        raise ValueError("YOU GET NOTHING")
+    
+    # Logic
+    char_position = lowecase_chars.index(character_argument)
+    
+    return return_message.format(char_position=char_position)
```

Comment:
okay this is the meat of the method but I think we can use ord here to simplify the whole thing. It will throw an exception if the string isn't a single character for us too so we could get rid of the previous sanity check too.

something like:
```suggestion
    # Logic
    return return_message.format(char_position=ord(character_argument) - 96)
```

Reply:
Isn't hardcoding -96 a bad pattern, Move to var?

Reply:
that'd be a fine distinction I guess it is a bit of a magic number without any context. you could extract it to some constant like `CHAR_OFFSET` or maybe just express it in a way that makes it clear what you're doing `(ord('a')-1)`